### PR TITLE
Dev/ireida/vuetify

### PR DESCRIPTION
**Pull request by ismailrei**

### CODE REVIEW

File: director/templates/index.html [hunk 1]
```diff
@@ -272,11 +277,12 @@
...
-    <script src="https://cdn.jsdelivr.net/npm/vuetify@2.2.1/dist/vuetify.min.js" integrity="sha256-Wksm4daF3JKKD97f/NHSJkxqQCk9XwujIEBTX0VLzTw=" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vuetify@2.3.8/dist/vuetify.min.js"  integrity="sha256-b4RV6u+xflpPubfyN5gdooQRpDjROUcaCCSWbKQNX9Y=" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/moment@2.24.0/min/moment.min.js" integrity="sha256-4iQZ6BVL4qNKlQ27TExEhBN1HFPvAvAMbFavKKosSWQ=" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/axios@0.19.0/dist/axios.min.js" integrity="sha256-S1J4GVHHDMiirir9qsXWc8ZWw74PHHafpsHp5PXtjTs=" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/d3@5.16.0/dist/d3.min.js" integrity="sha256-Xb6SSzhH3wEPC4Vy3W70Lqh9Y3Du/3KxPqI2JHQSpTw=" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/dagre-d3@0.6.4/dist/dagre-d3.min.js" integrity="sha256-dPm4TA8Y9PY5q5mmtWMkRGOCMHJDKy34ZrxdbBGA9cs=" crossorigin="anonymous"></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
```
Comment (anthonyolea) [inline director/templates/index.html:285]: If you want to add a link here, you need to use:
```
<script src="..." integrity="sha256-..." crossorigin="anonymous"></script>
```

File: director/templates/index.html [hunk 2]
```diff
@@ -272,11 +277,12 @@
...
     {% else %}
     <script src="%% url_for('user.static', filename='vue.min.js') %%"></script>
     <script src="%% url_for('user.static', filename='vue-router.min.js') %%"></script>
```
Comment (anthonyolea) [inline director/templates/index.html:288]: You have to add the _non-CDN_ link (and to update the `dlassets` command):
```
<script src="%% url_for('user.static', filename='iconify.min.js') %%"></script>
```

File: director/commands/assets.py [hunk 3]
```diff
@@ -49,13 +49,13 @@
         None,
     ),
     (
-        "https://cdn.jsdelivr.net/npm/vuetify@2.2.1/dist/vuetify.min.css",
-        "sha256-Ch5FMlNCdr6sBrwjHdGoTnAbAcKzE+UWiCXZHDT1HYk=",
+        "https://cdn.jsdelivr.net/npm/vuetify@2.3.8/dist/vuetify.min.js",
+        "sha256-b4RV6u+xflpPubfyN5gdooQRpDjROUcaCCSWbKQNX9Y=",
         None,
     ),
     (
```
Comment (anthonyolea) [inline director/commands/assets.py:56]: IMO, you have to add the `iconify.min.js` (v1.0.7) file link here (and related sha-256).

File: director/templates/index.html (hunk 1 above)
Comment (ismailrei) [reply director/templates/index.html]: Done

File: director/commands/assets.py [hunk 4]
```diff
@@ -24,8 +24,8 @@
...
-        "https://cdn.jsdelivr.net/npm/vuetify@2.2.1/dist/vuetify.min.js",
-        "sha256-Wksm4daF3JKKD97f/NHSJkxqQCk9XwujIEBTX0VLzTw=",
+        "ttps://cdn.jsdelivr.net/npm/vuetify@2.3.8/dist/vuetify.min.css",
```
Comment (ncrocfer) [inline director/commands/assets.py:27]: `tts`:)

**Review by anthonyolea — APPROVED** [review]
LGTM

**Review by ncrocfer — APPROVED** [review]

**Pull request merged**